browser_patches/webkit/patches/bootstrap.diff

```diff
@@ -17123,11 +17135,8 @@ index 0000000000000000000000000000000000000000..94e9f24353337169992724e2fcdf7086
 +
 +private:
 +    void platformDispatchKeyEvent(WebKeyboardEvent::Type type, const String& text, const String& unmodifiedText, const String& key, const String& code, const String& keyIdentifier, int windowsVirtualKeyCode, int nativeVirtualKeyCode, bool isAutoRepeat, bool isKeypad, bool isSystemKey, OptionSet<WebEvent::Modifier> modifiers, Vector<String>& commands, WallTime timestamp);
-+#if PLATFORM(WPE)
-+    void platformDispatchMouseEvent(WebMouseEvent::Type type, int x, int y, WebMouseEvent::Button button, OptionSet<WebEvent::Modifier> modifiers);
```

Comment:
Somehow this split was important.

Reply:
Maybe it was, but this method is not implemented anywhere and never called.